assets/_sass/components/_social-share-icons.scss

```diff
@@ -0,0 +1,69 @@
+/*==================================
+=       Social Share Icons         =
+==================================*/
+
+.share {
+    $s: &;
+    $i: icon-facebook, icon-twitter, icon-email, icon-print;
+    display: flex;
+    padding: .5rem;
+
+    &__text {
+        @include btn;
+        text-transform: uppercase;
+        padding-bottom:.2rem;
+    }
+
+    ul {
+        list-style-type: none;
+        display: flex;
+        justify-content: space-between;
+        margin: 0;
+        padding: 0;
+
+        li {
+            display: inherit;
+            margin: 0;
+            padding-right: 0.1rem;
+            text-align: center;
+            line-height: 1;
+
+            i {
+                font-size: 1.3rem;
+            }
+        }
+    }
+
+    .page-header & {
+        background-color: $color-dust;
```

Comment:
Don't worry about setting the background here or on line 54. The background will be set by the parent component.